roles/servicetelemetry/tasks/main.yml

```diff
@@ -12,9 +12,15 @@
   set_fact:
     has_interconnect_api: "{{ True if 'interconnectedcloud.github.io' in api_groups else False }}"
 
+- name: Check if we have certmanager.k8s.io API
+  set_fact:
+    has_certmanager_api: "{{ True if 'certmanager.k8s.io' in api_groups else False }}"
+
 - name: Create QDR instance
   include_tasks: component_qdr.yml
-  when: has_interconnect_api | bool
+  when:
+    - has_interconnect_api | bool
+    - has_certmanager_api | bool
```

Comment:
This might help with a minor annoyance I found in dev when forgetting to deploy the certmanager

Reply:
I wonder if adding some sort of debugging information with a sort of "error" style warning would be useful here in the logs? I could add an `ignore_errors: true` on a log output with a warning style level.

I won't do it for this PR (it's already about 4 times larger than I expected), but I think it might be worth having some of those checks and missing interfaces called out a bit more strongly in the console output.